booklist.js

```diff
@@ -0,0 +1,28 @@
+export default class BookList {
+  constructor() {
+    this.list = [];
+  }
+
+  addBook(book) {
+    this.list.push(book);
+  }
+
+  removeBook(book) {
+    this.list = BookList.arrayRemove(this.list, book);
+  }
+
+  static arrayRemove(arr, book) {
+    return arr.filter(
+      (ele) => ele.title !== book.title && ele.author !== book.author,
+    );
+  }
```

Comment:
- [ ] Let's address one issue please. If by chance there are two books and supposedly of the same title, calling the `removeBook` method will delete all these books instead of just one. It is a requirement that when we delete a book, only the intended book will be deleted whatever the case.

## Possible remedies:
- Filter the array by book index when removing a book
- Prevent addition of duplicates
- [OPTIONAL] add a unique `id` property to a book object and filter the array of books using the id when removing a book

Reply:
This issue is well resolved ✔️